### PR TITLE
Create Infoblox IB-825

### PR DESCRIPTION
**Pull request by Lon1**

### CODE REVIEW

File: device-types/Infoblox/IB-825.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Infoblox
+model: IB-825
+slug: infoblox-ib-825
+is_full_depth: false
```
Comment (danner26) [inline device-types/Infoblox/IB-825.yaml:5]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not. I believe the airflow is `front-to-rear`
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not. I believe the weight is `7.71 kg`
3. Please add the manufacturer `part_number` which I believe is `TE-825`

If you have any questions, please let us know :)

File: device-types/Infoblox/IB-825.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Infoblox
+model: IB-825
```
Comment (danner26) [inline device-types/Infoblox/IB-825.yaml:3]: Is the device actually called `Trinzic 825`?

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Infoblox/IB-825.yaml (hunk 2 above)
Comment (Lon1) [reply device-types/Infoblox/IB-825.yaml:3]: Trinzic would be the original name of the company from long ago before it was Infoblox.  I go by what our NetMRI tool discovers it as (shown below).  NetMRI is a Network Management platform from Infoblox.
image

Comment (Lon1) [reply device-types/Infoblox/IB-825.yaml:3]: Daniel - I took a look at what our actual product sheet said for these devices and it uses TE-805-HW-AC.  That is the base hardware.  The '2' (in the 825) is addition software options.  Splitting hairs here, but wanted to be fair on what direction you wanted to go with this.

image

Comment (danner26) [reply device-types/Infoblox/IB-825.yaml:3]: So the hardware is what we are modelling here. I think it is best to make the part number equal what the hardware is and then have the model number reflect IB-825

Comment (Lon1) [reply device-types/Infoblox/IB-825.yaml:3]: I would agree to that.  I'll try and update the others with the part number as well next week.

File: device-types/Infoblox/IB-825.yaml (hunk 1 above)
Comment (Lon1) [reply device-types/Infoblox/IB-825.yaml:9]: added requested fields.

**Review by Lon1 — COMMENTED** [review]
added requested fields

**Review by danner26 — APPROVED** [review]

**Pull request merged**